contrib/net/http/option.go

```diff
@@ -108,6 +109,13 @@ func WithAfter(f RoundTripperAfterFunc) RoundTripperOption {
 	}
 }
 
+// RTWithServiceName sets the given service name for the RoundTripper
+func RTWithServiceName(name string) RoundTripperOption {
```

Comment:
Such a terrible name! I guess we did this to ourselves 🙁 I opened https://github.com/DataDog/dd-trace-go/issues/532 to not forgot and improve this somehow (even if not in the way proposed) in v2.

Reply:
Yeah, I have been thinking about ways to do this better. There is also #493